server/app/services/account-service.ts

```diff
@@ -515,6 +515,35 @@ const setGlobalPermissions = async (
   }
 };
 
+const updateUserProfile = async (
+  userid: string,
+  firstName: string,
+  lastName: string,
+  email: string
+) => {
+  const id: number = Number(userid);
+  const sql = `update login
+                set first_name = $<firstName>,
+                last_name = $<lastName>,
+                email = $<email>
+                where id=$<id>`;
+  try {
+    await db.none(sql, { id, firstName, lastName, email });
+
+    return {
+      isSuccess: true,
+      code: "UPDATE_SUCCESS",
+      message: `User profile successfuly updateed, first_name: ${firstName},  last_name: ${lastName},  and email: ${email}`,
```

Comment:
`successfully updated` 😅

Reply:
I think we also want to return the new user so we can update the user context 🤔 Currently, it's getting updated in the db but the ui isn't getting synced because we're not updating the user context. We probably also need to update the user in session storage 😅 

https://user-images.githubusercontent.com/25972384/229321322-2a10459f-ee97-491e-9020-b3bc178546b1.mov